libherokubuildpack/src/buildpack_output/background_printer.rs

```diff
@@ -0,0 +1,195 @@
+//! This module is responsible for the logic involved in the printing to output while
+//! other work is being performed. Such as printing dots while a download is being performed
+use std::io::Write;
+use std::sync::mpsc::{channel, Sender};
+use std::thread::JoinHandle;
+use std::time::Duration;
+
+/// Print dots to the given buffer at the given interval
```

Comment:
Printing dots is given as an example of how this can be used but, for this method, there are no dots. 

```suggestion
/// Repeatedly prints `tick` to the given buffer at the given interval. The `start` argument will be printed before the first `tick` and the `end` argument will be printed after the last `tick` when the timer is stopped.
```